### ISSUE TITLE
How to disable caching?

### ISSUE THREAD

**Issue by holms** [jdxcode/npm-register#127]
I've actually encoutered error with this:
```
 npm install react --save-peer
 npm install react-native --save-peer
```

It fails to find a package and I actually have no idea how to investigate it, without registry it works.
So I'd like simply to disable caching, how can I do this?

**Comment by holms** [comment]
```
holms@debian ~/D/c/l/a/app-account> npm install
npm WARN deprecated istanbul-lib-hook@1.2.1: 1.2.0 should have been a major version bump
npm WARN deprecated socks@1.1.10: If using 2.x branch, please upgrade to at least 2.1.6 to avoid a serious bug with socket data flow and an import issue introduced in 2.1.0
npm ERR! code E404
npm ERR! 404 Not Found: axios@^0.18.0

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/holms/.npm/_logs/2018-08-31T11_16_41_230Z-debug.log
```

Here you go, and there's nothing I can do

**Comment by dgautsch** [comment]
@holms found the issue, for some reason the response on the axios package details from npm is not coming back as an object but a string. Probably because it's so big. I'm putting a check in to parse that out. See ticket #129

**Comment by dgautsch** [comment]
@holms with v.2.8.1 are you still encountering this?

**Comment by dgautsch** [comment]
Closing as this appears to be fixed by v2.8.1 #129
